docs/hello-sample.md

```diff
@@ -1,150 +1,300 @@
 
 ## Get it and build it
 
-The sample is in the directory [/samples/hello](https://github.com/microsoft/durabletask-netherite/tree/main/samples/Hello). You can download just this folder if you wish, or clone the entire repository. Note that PowerShell sometimes refuses to execute downloaded files; so using git clone may work better.
+The samples are in the directory [/samples/](https://github.com/microsoft/durabletask-netherite/tree/main/samples). You can download just this folder if you wish, or clone the entire repository. Note that PowerShell sometimes refuses to execute downloaded files so using git clone may work better.
+
+The language-specific folder in the `samples` folder contain "minimal" Azure Durable Functions projects (much like the [DF Quick Start](https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-create-first-csharp?pivots=code-editor-visualstudio)).
+
+⚠ **CAUTION:** Be aware that all projects have no authentication in place. This makes the calling of the Functions easy, but this is not feasible setup for a production environment.
+
+The build process depends on the language you want to use. The following sections guide you through the specific steps.
+
+### C#
+
+The folder [/samples/Hello_Netherite_with_DotNetCore](https://github.com/microsoft/durabletask-netherite/tree/main/samples/Hello_Netherite_with_DotNetCore) contains the files for the Durable Function App in C#.
+
+The main files amd folders are:
 
-The project contains a "minimal" .NET Azure Durable Functions project (much like the [DF Quick Start](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-create-first-csharp?pivots=code-editor-visualstudio)):
-- a single file of code, `HelloCities.cs`, which defines three functions: a trigger, an orchestration, and an activity.
-- a configuration file `host.json` which contains settings.
-- a configuration file `local.settings.json` which contains settings for local testing.
+- the single file of code, `HelloCities.cs`, which defines three functions: a trigger, an orchestration, and an activity.
+- the configuration file `host.json` which contains settings.
+- the configuration file `local.settings.json` which contains settings for local execution.
 
-Additionally, the folder contains six scripts (files ending in *.ps1) that make it easy to perform typical operations (init, run, deploy, clear, delete) using automated CLI commands. These scripts are purely optional; one can perform all the steps in these scripts manually, using the Azure CLI or the Azure Portal.
+Additionally, the folder contains PowerShell scripts (files ending in `*.ps1`) that make it easy to run amd deploy the Function using automated CLI commands. These scripts are optional. You can perform all the steps in these scripts manually, using the Azure CLI or the Azure Portal.
```

Comment:
nit: amd -> and

Reply:
Corrected ... there have been several "amd" in there thanks to copy&paste ;-)